js-client/packages/pipe/src/lib/pipe.ts

```diff
@@ -14,10 +14,10 @@ export class Pipe {
     #connection$
     #subscriptions: SubscriptionState[] = []
 
-    constructor({ url }: { url: string }) {
+    constructor({ url, options }: { url: string; options?: IHttpConnectionOptions }) {
         this.#connection$ = new Observable<HubConnection>(subscriber => {
             const connection = new HubConnectionBuilder()
-                .withUrl(url)
+                .withUrl(url, options ?? {})
```

Comment:
is `?? {}` necessary?

Reply:
it is, `withUrl` doesn't allow `undefined` as args